portfolio/src/main/webapp/comment-form.js

```diff
@@ -0,0 +1,20 @@
+// Retrieve and post user comments
```

Comment:
These are documentation comments, not implementation, so JSDoc should be used

https://google.github.io/styleguide/jsguide.html#jsdoc

Reply:
I went through each of my JS files and switched documentation from // to /** */